.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
   global:
     - RAILS_ENV=test
     - CC_TEST_REPORTER_ID=f39e060a8b1a558ebd8aff75d5b9760bf1ae98f3f85d628ae28814f3c66438cd
+    - DATABASE_URL=postgres://postgres@localhost/
```

Comment:
Oh, I wish Travis doc stated this. It's so much more intuitive to do it this way.

Reply:
I've never been a super big fan of Travis docs.